src/main/java/com/archisacadeny/student/StudentService.java

```diff
@@ -0,0 +1,39 @@
+package com.archisacadeny.student;
+
+import java.util.List;
+
+public class StudentService {
+    private final StudentRepository studentRepository;
+
+
+    public StudentService(StudentRepository studentRepository) {
+        this.studentRepository = studentRepository;
+    }
+
+    public Student createStudent(Student student) {
+
```

Comment:
please fill the method bodies with more details for instance printing the data and etc